youtube_dl/extractor/makotv.py

```diff
@@ -0,0 +1,62 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import (
+    update_url_query,
+    urljoin
+)
+
+
+class MakoTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?mako\.co\.il/mako-vod-.+/VOD-(?P<id>[0-9a-f]{18})\.htm'
+    _TEST = {
+        'url': 'https://www.mako.co.il/mako-vod-keshet/parliament-s1/VOD-5df5a86c1966831006.htm',
+        'md5': 'cd8cdff75390f8521831ec5049841764',
+        'info_dict': {
+            'id': '5df5a86c1966831006',
+            'ext': 'm3u8',
+            'title': 'הפרלמנט | פרק 1 לצפייה ישירה | makoTV ',
```

Comment:
Must only contain title.

Reply:
This is how the title appears in the `<title>` tag, in the `content` of the `<meta property="og:title">` tag, and in the `"name"` of the `<script type="application/ld+json">` tag.
I think the best I can do is to remove the "makoTV" suffix with a regex or something.
What do you think?